articles/documentation-docfx.md

```diff
@@ -2,33 +2,33 @@
 
 Effective online documentation plays a crucial role in driving the adoption and usage of custom packages. For packages published by the Bonsai Foundation, we use [docfx](https://dotnet.github.io/docfx/index.html), a static site generator for .NET projects, to automatically generate and publish documentation online. As we have integrated and extended `docfx` to facilitate key Bonsai features, we encourage package developers to use `docfx` for consistency and support within the Bonsai Ecosystem. 
 
-The following article will go into how to create a `docfx` website for your package. For documentation style guidelines, check out the [Documentation Style Guide](./documentation-style-guide.md).
+The following article will go into detail on how to create a `docfx` website for your package. For documentation style guidelines, check out the [Documentation Style Guide](./documentation-style-guide.md).
 
 > [!TIP]
 > We recommend using a code editor with an integrated terminal like [Visual Studio Code](https://code.visualstudio.com/) to follow along with these instructions.
 
 ## Repository organization
-For effective long term maintenance of documentation across different repositories, package repositories should be organised according to this standardized folder structure before setting up `docfx`: 
+For effective long term maintenance of documentation, package repositories should be organized according to the following standard folder structure before setting up `docfx`: 
 
 ```
 .
 ├── .bonsai/                 # Local Bonsai Environment (see note below) 
-├── .github/workflows/       # Github Actions Folder (see note below) 
+├── .github/workflows/       # GitHub Actions Folder (see note below) 
 │   └── docs.yml
 ├── docs/                    # Folder to setup docfx
 └── src/
     └── Bonsai.PackageName/  # Project files here
 ... 
 ```
 
-- `.bonsai/` - A local Bonsai [environment](https://bonsai-rx.org/docs/articles/environments.html) is required for a script that exports Bonsai workflow images. To create one run these following commands:
+- `.bonsai/` - A local Bonsai [environment](https://bonsai-rx.org/docs/articles/environments.html) is required to run the script that exports Bonsai workflow images. To create one, run the following commands:
```

Comment:
```suggestion
- `.bonsai/` - A local Bonsai [environment](./environments.md) is required to run the script that exports Bonsai workflow images. To create one, run the following commands:
```

Noticing now this and a few other links could be replaced with local markdown references, useful when debugging local docs preview.